src/hooks/sort-by.js

```diff
@@ -12,18 +11,39 @@ export default class extends BaseHooks {
 
         this.sortByEvents();
     }
+    // ie 11 does not support native closest so use this function instead
+    closest(el, selector) {
+        let element = el;
+        do {
+            if (element.matches(selector)) {
+                return element;
+            }
+            element = element.parentElement || element.parentNode;
+        } while (element !== null && element.nodeType === 1);
+        return null;
+    }
 
     sortByEvents() {
-        this.$body.on('submit', '[data-sort-by]', (event) => {
-            this.emit('sortBy-submitted', event);
-        });
-
-        this.$body.on('change', '[data-sort-by] select', (event) => {
-            this.emit('sortBy-select-changed', event);
+        const sortByElements = document.querySelectorAll('[data-sort-by]');
+        if (sortByElements.length) {
+            for (let i = 0; i < sortByElements.length; i++) {
+                sortByElements[i].addEventListener('submit', (event) => {
+                    this.emit('sortBy-submitted', event);
+                });
+            }
+        }
+        const selectedElements = document.querySelectorAll('[data-sort-by] select');
+        if (selectedElements.length) {
+            for (let i = 0; i < selectedElements.length; i++) {
+                selectedElements[i].addEventListener('change', (event) => {
+                    this.emit('sortBy-select-changed', event);
 
-            if (! event.isDefaultPrevented()) {
-                $(event.currentTarget).closest('form').trigger('submit');
+                    if (!event.defaultPrevented) {
+                        // ie11 does not support native closest
+                        this.closest(event.currentTarget, 'form').submit();
```

Comment:
What does `this` refer to here? Is the matching element, or the default class?

Reply:
default class.